presto-main/src/main/java/com/facebook/presto/dispatcher/FailedDispatchQuery.java

```diff
@@ -115,7 +115,10 @@ public void fail(Throwable throwable) {}
     public void cancel() {}
 
     @Override
-    public void pruneInfo() {}
+    public void pruneQueryExpired() {}
+
+    @Override
+    public void pruneQueryFinished() {}
```

Comment:
How about this naming? 

```suggestion
    public void pruneExpiredQueryInfo() {}

    @Override
    public void pruneFinishedQueryInfo() {}
```

Reply:
updated